hw/ip/otbn/doc/dv/index.md

```diff
@@ -414,7 +435,8 @@ This instruction uses the `bnaf` encoding schema, with covergroup `enc_bna_cg`.
 
 This instruction uses the `bnai` encoding schema, with covergroup `enc_bnai_cg`.
 
-Specific points as for `BN.ADD`.
+- Extremal values of shift for both directions where the shifted value is nonzero
```

Comment:
Shift? This is ADDI. Perhaps a copy-paste error.

Reply:
This is BN.ADDI, which has a shift operand. Read [the ISA doc](https://docs.opentitan.org/hw/ip/otbn/doc/isa/#bnaddi) for more information.